experimenter/experimenter/templates/experimenter_base.html

```diff
@@ -8,9 +8,32 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>{% block title %}{% endblock %}</title>
     <script src="{% static 'fontawesomefree/js/all.min.js' %}"></script>
+    <script src="https://cdn.tailwindcss.com"></script>
 </head>
 <body>
-    {% block content %}
-    {% endblock %}
+    <header class="w-full bg-white sticky top-0 left-0 px-8 py-2 flex justify-center items-center shadow-md shadow-[#b590d4]">
+        <div class="max-w-screen-2xl w-full flex justify-between items-center space-x-4">
+            <div class=" flex justify-start items-center space-x-4">
+                <img class="w-12" src="{% static 'assets/logo.svg' %}" />
+                <h1 class="text-2xl text-[#242D4Eff]">Nimbus Experiments</h1>
+            </div>
+            <div class="lg:flex text-[#242D4Eff] justify-center items-center hidden">
+                <div class="flex  justify-start items-center space-x-6">
+                    <a href="https://experimenter.info/" target="_blank" rel="noreferrer">Docs</a>
+                    <a href="https://mozilla-hub.atlassian.net/secure/CreateIssueDetails!init.jspa?pid=10203&issuetype=10097" target="_blank" rel="noreferrer">Feedback</a>
+                    <a href="https://mozilla.slack.com/?redir=%2Farchives%2FCF94YGE03" target="_blank" rel="noreferrer">Help</a>
+                </div>
+            </div>
+        </div>
+    </header>
+    <div class="h-full min-h-screen w-full flex flex-col justify-start items-center">
+        {% block content %}
+        {% endblock %}
+    </div>
+    <footer class="w-full mt-2 flex justify-center items-center">
+            <div class="flex flex-col justify-center text-[#242D4Eff] items-center w-5/6 p-4 border-t border-[#b590d4]">
+              © Mozilla Corporation 2023
+            </div>
```

Comment:
would suggest creating two different header and footer and then import into base